docs/whats_new.rst

```diff
@@ -12,9 +12,13 @@ Release |version|
 Powered by Miniconda Environment
 --------------------------------
 
-TODO
+* Tethys Platform is now installed in a Miniconda environment.
+* Using the Miniconda includes Conda, an open source Python package management system
+* Conda can be used to install Python dependencies as well as system dependencies
+* Installing packages like GDAL or NetCDF4 are as easy as ``conda install gdal``
```

Comment:
Should we just add conda-forge as the default channel in the install script? Either way some documentation about channels would be useful.